config/routes.rb

```diff
@@ -13,5 +13,17 @@
 
   resources :posts, only: [:index, :show, :destroy]
 
+  namespace :api do
+    namespace :v1 do
+      resources :users, only: [:index, :show] do
+        resources :posts, only: [:index]
+      end
+
+      resources :posts do
+        resources :comments, only: [:index, :create]
```

Comment:
**OPTIONAL** - You did well so far, but to make your code more readable and easier to understand, I suggest you nest your comment route under the `post` resource above. Please fix this 👍 👍 

```ruby
   namespace :api do
    namespace :v1 do
      resources :users, only: [:index, :show] do
        resources :posts, only: [:index] do
           resources :comments, only: [:index, :create]
        end
      end

    end
  end